src/components/Filter.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { FilterOptions } from '../types/FilterOptions';
+import classNames from 'classnames';
+import { useTodos } from '../utils/TodoContext';
+
+const filterOptionsArray = Object.values(FilterOptions);
+
+export const Filter: React.FC = () => {
+  const { filter, setFilter } = useTodos();
+
+  return (
+    <nav className="filter" data-cy="Filter">
+      {filterOptionsArray.map(option => (
+        <a
+          key={option}
+          href="#/"
```

Comment:
`href` too should be change with 'option' value:
```suggestion
          href={`#${option}`}
```